code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -769,16 +769,16 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	else
 		L = get(src, /mob/living/silicon)
 
-	if(L)
+	if(L && L.stat != UNCONSCIOUS)

```

Comment:
Yeah I just meant that it's information they can access either way, so it's not that big of a deal.